gubernator/regex.py

```diff
@@ -16,12 +16,26 @@
 
 import re
 
-# Matches UID from the object reference 
+# Match UID from the object reference 
 uidobj_re = re.compile(r'Event\(api\.ObjectReference\{[^}].*UID:&#34;(.*?)&#34;(, [^}]*)?\}')
 
-# Matches a specific word
+# Match a specific word
 def wordRE(word):
 	return re.compile(r'\b(%s)\b' % word, re.IGNORECASE)
 
-# Matches against words suggesting an error
-errors_re = re.compile(r'\b(error|fatal|failed|build timed out)\b', re.IGNORECASE)
\ No newline at end of file
+# Match lines with error messages
+error_re = re.compile(r'\b(error|fatal|failed|build timed out)\b', re.IGNORECASE)
+
+# Match the keys in the object reference string
+keys_re = re.compile(r'[\s|\{](.*?):')
```

Comment:
nit: prefer negated character classes to lazy matchers, e.g. `[^:]*` over `.*?`. (See "An alternative to laziness" at http://www.regular-expressions.info/repeat.html for an explanation for an explanation)